sailfishusermanagerinterface.h

```diff
@@ -15,7 +15,7 @@
 
 #define SAILFISH_USERMANAGER_DBUS_INTERFACE "org.sailfishos.usermanager"
 #define SAILFISH_USERMANAGER_DBUS_OBJECT_PATH "/"
-#define SAILFISH_USERMANAGER_MAX_USERS 7
+#define SAILFISH_USERMANAGER_MAX_USERS 6
```

Comment:
I would prefer to include master user. Or is there some compelling reason to count only additional users?

Reply:
I was thinking this in more of naming thing, user != master.  If objections I can revert this and drop the count in addUser too.

Reply:
Hmm. A call to users() returns also master user, right? That's why I'd think this count should also include master user. But I mean as long as it's clear and documented I'm okay with it. Currently I think it's quite confusing.

Reply:
Fixed